README.md

```diff
@@ -1 +1,55 @@
-# Hello-World
\ No newline at end of file
+<a name="Bhekisile Jozi"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Hello Microverse]** is a project that teaches about how to successfully use linters on HTML and CSS files as well to make a file disregarded by git and not present in your GitHub repository.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### HTML and CSS <a name="tech-stack"></a>
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[showing text]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Bhekisile Jozi**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
```

Comment:
- [ ] As the last code reviewer said, all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in your repo, please 🙏🏽 complete your README file with all sections: 👇🏽 

> Every project is expected to be properly documented with details including how other programmers can run the project on their machines. As stated in the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/083a7b724f96d81883401e8d6602aeac363301d4/documentation/hello_microverse_project.md?plain=1#L40), kindly use [this template](https://github.com/microverseinc/readme-template) by Microverse to include a descriptive readme for your project, customizing the sections to match your current project.